Tasks/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -54,6 +50,49 @@ async function run(): Promise<void> {
             tl.debug(tl.loc("Info_CreatingSettingsXml", userSettingsXmlPath));
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
+        
+#if WIF
+        const feedUrl = tl.getInput("feedUrl");
+        const entraWifServiceConnectionName = tl.getInput("workloadIdentityServiceConnection");
+        const feedName = tl.getInput('feedName');
+
+        if (feedUrl && entraWifServiceConnectionName && feedName) {
```

Comment:
As we discussed for npm, we should fail if some is not set.

Reply:
Changed this!